R/adjust-numeric-range.R

```diff
@@ -61,28 +61,20 @@ adjust_numeric_range <- function(x, lower_limit = -Inf, upper_limit = Inf) {
 #' @export
 print.numeric_range <- function(x, ...) {
   trn <- ifelse(x$trained, " [trained]", "")
-  # todo could be na
```

Comment:
Technically this works as if, since `signif(NA) = NA`. it will thus show up as `NA` in the print method.

If we want it to show `missing` that would't be hard to change